gradle/dependenciesCheck.gradle

```diff
@@ -21,8 +21,20 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'org.owasp:dependency-check-gradle:3.0.1'
+        classpath 'org.owasp:dependency-check-gradle:3.0.2'
     }
 }
 
-apply plugin: org.owasp.dependencycheck.gradle.DependencyCheckPlugin
+configure(projectsIncludedInDistribution()) {
+    apply plugin: org.owasp.dependencycheck.gradle.DependencyCheckPlugin
+
+    dependencyCheck {
+        failBuildOnCVSS = 8 // 10 is the maximum
```

Comment:
⭕️ "10 is the maximum" doesn't really tell me what this means. Do the numbers have specific meanings or are they arbitrarily assigned?

Reply:
The Common Vulnerability Scoring System (CVSS) uses several aspects of exploit-ability and impact to determine a base score. Each vulnerability in the NVD has a base score attached, and anything above 4.0 is considered Medium or High severity ([CVSS overview](https://nvd.nist.gov/vuln-metrics/cvss)).

It's a calculated value. The value 10 indicates Critical severity. So with the value 8 we didn't score so well. In a follow up issue we should try to get down the number.